services/ui-src/src/utils/other/export.test.tsx

```diff
@@ -53,6 +56,56 @@ describe("Test rendering methods", () => {
     );
     expect(result.props.children.id).toEqual("email-field-id");
   });
+
+  test("renderDrawerDataCell renders ilos responses", () => {
+    const mockFormField: FormField = {
+      id: "plan_ilosUtilizationByPlan",
+      props: {
+        choices: [],
+      },
+      type: "checkbox",
+      validation: "checkbox",
+    };
+
+    const mockPlan: AnyObject = [
+      {
+        id: "mock-id",
+        plan_ilosUtilizationByPlan: [
+          {
+            key: "123",
+            value: "mock-ilos",
+          },
+        ],
+        plan_ilosUtilizationByPlan_123: "N/A",
+      },
+    ];
+
+    const cells = renderDrawerDataCell(mockFormField, mockPlan, "drawer");
+    const ul = cells[0].props.children;
+    const li = Array.from(ul.props.children)
+      .flat()
+      .filter((el) => (el as ReactJSXElement)?.type === "li");
+    expect(cells.length).toBe(1);
+    expect(li.length).toBe(3);
```

Comment:
After thinking about it, it seemed there should be a way to use `render` here and I think this is cleaner:

```suggestion
    const cells = renderDrawerDataCell(mockFormField, mockPlan, "drawer");
    const Component = () => cells[0];
    const { container } = render(<Component />);
    expect(container.querySelectorAll("li").length).toBe(3);
    expect(container.querySelectorAll("li")[2]).toHaveTextContent("mock-ilos: N/A");
```